### PR TITLE
fixed positioning of fb & twitter icons

### PR DESCRIPTION
**Pull request by JacobStewart20**
| Before | After |
|--------|-------|
| Screenshot 2019-10-07 at 17 15 37 | Screenshot 2019-10-07 at 17 15 42 |

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,153 +1,164 @@
...
+/* better positions facebook and twitter icons */
+.fa-facebook:before {
+  content: "\f09a";
+  vertical-align: inherit;
+}
+
+.fa-twitter:before {
+  content: "\f099";
+  vertical-align: inherit;
+}
```
Comment (jenkoian) [inline src/style.css]: Not sure what's going on here, why does github think the entire file has changed? Just a github quirk? Either way, can you add a newline at the end of the file please?

Comment (JacobStewart20) [reply src/style.css]: Yeah not sure why it says the entire file has changed, only added to the end of it.  & yes I'll add that!

**Comment by jenkoian** [comment]
@JacobStewart20 you ok to resolve conflicts?

**Comment by JacobStewart20** [comment]
Of course! Done. @jenkoian

File: src/style.css [hunk 2]
```diff
@@ -140,8 +140,7 @@ a:hover {
   background: inherit;
 }
 
-.pull-request,
-.faqs {
+.pull-request {
```
[the remark this answers is not in the document]
Comment (JacobStewart20) [reply src/style.css]: Oh, yeah that was not intended. I messed up my pull request. I am sorry! However, it has been corrected. & yes I ran that command to generate the index.css. Should be all good now, sorry for all of the back of forth little stuff. @jenkoian

**Comment by jenkoian** [comment]
Thanks @JacobStewart20 !

**Pull request merged**